src/Authenticator/LoginHandler.php

```diff
@@ -666,7 +570,7 @@ protected function doPerformLogin(HTTPRequest $request, Member $member)
             // These next two lines are pulled from "parent::doLogin()"
             $this->performLogin($member, $data, $request);
             // Allow operations on the member after successful login
-            $this->extend('onLoginSuccess', $member);
+            parent::extend('afterLogin', $member);
```

Comment:
For public visibility, here's a summary of what we discussed internally with this change:

* We can't delegate to the parent method on success, because the member data is gone at this point. We overload it instead.
* The `afterLogin` hook comes from the parent class, and I removed it thinking we didn't need it in #103.
* The `onLoginSuccess` hook is something we added and don't actually need.
* We're asking the parent to express the extension hook, because people may have listeners attached to the parent class rather than this child.